light-zk.js/src/merkleTree/merkleTreeConfig.ts

```diff
@@ -530,7 +530,7 @@ export class MerkleTreeConfig {
     if (!this.merkleTreeAuthorityPda)
       throw new Error("merkleTreeAuthorityPda undefined");
 
-    let registeredPoolTypePda = this.poolTypes.filter((item, index) => {
+    let registeredPoolTypePda = this.poolTypes.filter((item, _index) => {
```

Comment:
if _index is unused, why bother typing it out? we can just do ((item) => ...)

Reply:
sure, same as the other comment, was unsure since is not my code

Reply:
feel free to remove it since it's not used

Reply:
removed ✅